packages/atlas/src/views/viewer/VideoView/VideoView.tsx

```diff
@@ -61,25 +76,19 @@ export const VideoView: React.FC = () => {
   }, [video, thumbnailUrl])
   const headTags = useHeadTags(video?.title, videoMetaTags)
 
-  const [startTimestamp, setStartTimestamp] = useState<number>()
+  const { startTimestamp, setStartTimestamp } = useVideoStartTimestamp(timestampFromQuery, video?.duration)
```

Comment:
I think we should be fine with including the hook that gets value of `timestampFromQuery` inside this hook as well?